language/analysis/src/test/java/com/google/cloud/language/samples/AnalyzeTest.java

```diff
@@ -69,13 +71,22 @@
     PrintStream out = new PrintStream(bout);
     ImmutableList<Entity> entities =
         ImmutableList.of(
-            new Entity().setName("Larry Page").setSalience(0.426f).setType("PERSON").setMetadata(
-                ImmutableMap.<String, String>builder()
-                    .put("knowledge_graph_mid", "/m/0gjpq")
-                    .put("wikipedia_url", "http://en.wikipedia.org/wiki/index.html?curid=60903")
-                    .build()),
-            new Entity().setName("search engine").setSalience(0.188f).setType("CONSUMER_GOOD"),
-            new Entity().setName("something"));
+            Entity.newBuilder().setName("Larry Page")
+                .setSalience(0.426f)
+                .setType(Type.PERSON)
+                .putAllMetadata(
+                    ImmutableMap.<String, String>builder()
+                        .put("knowledge_graph_mid", "/m/0gjpq")
+                        .put("wikipedia_url",
+                            "http://en.wikipedia.org/wiki/index.html?curid=60903")
+                        .build())
+                .build(),
+            Entity.newBuilder()
+                .setName("search engine")
+                .setSalience(0.188f)
```

Comment:
Done. Unless, you mean to comment on the inner members, e.g.

    Entity.newBuilder()
        .setName("search engine") // The proper name for the mock entity
        .setSalience(0.188f) // The prominence of this entity relative to others in the document

I'm not sure how much detail is appropriate here in the mocks. For example, AFAICT the salience of entities sums up to 1. This means that if you sorted the entities by salience, descending, the most prominent entities in the target text will appear first: the actual salience values just tell you how important this particular entity is within the scope of the document.